db/seeds.rb

```diff
@@ -12,5 +12,3 @@
   Doorkeeper::Application.create(name: "iOS client", redirect_uri: "", scopes: "")
   Doorkeeper::Application.create(name: "Android client", redirect_uri: "", scopes: "")
 end
-
-Fabricate(:user)
```

Comment:
Is it rebase conflict issue?

Reply:
@junan While testing again with new db, I realize this is not needed (now that I better understand how tests works). So I removed it inside this branch as it's still related to authentication.